docs/errors/E0710.md

```diff
@@ -0,0 +1,23 @@
+# E0710: '^' is the XOR operator; to exponentiate, use '**' instead
+
+The Exclusive OR operator ^ can sometimes be mistaken for the
+exponentiation operator **, especially when applied to literal bases 2 and 10
+and literal exponent operations (which could be written as literal itself)
```

Comment:
Nit: I think the second clause of this sentence makes the whole statement more confusing than it needs to be. I'd remove it.
```suggestion
exponentiation operator **.
```